rmgpy/solver/base.pyx

```diff
@@ -456,6 +458,34 @@ cdef class ReactionSystem(DASx):
         
         return
     
+    cpdef addReactionsToSurface(self,list newSurfaceReactions,list newSurfaceReactionInds,list surfaceSpecies,list surfaceReactions,list edgeSpecies):
+        """
+        moves new surface reactions to the core
```

Comment:
The docstring doesn't seem to match the method name. Does this add  a reaction to the core or to the surface?